test/rest_api/test_compaction_task.py

```diff
@@ -12,10 +12,13 @@
 long_time = 1000000000
 
 # depth parameter means the number of edges in the longest path from root to leaves in task tree.
-def check_compaction_task(cql, this_dc, rest_api, run_compaction, compaction_type, depth, allow_no_children=False):
+def check_compaction_task(cql, this_dc, rest_api, run_compaction, compaction_type, depth, allow_no_children=False, disable_tablets=False):
```

Comment:
Looks like this test covers important vnodes-related functionality and with the current setup, it will never run.
I suggest the following:
Add a fixture called `test_keyspace_vnodes`, and make this test depend on it. This fixture does the following:
* if tablets are not enabled, just create  regular keyspace
* if tablets are enabled, creates a keyspace with tablets disabled on it (` AND TABLETS = {'enabled': 'false'}`)

Do this for all other tests that cover vnodes-related functionality.